assets/sass/v2/_forms.scss

```diff
@@ -11,6 +11,9 @@
   }
 
   .siw-main-body {
+    .o-form-content {
+      overflow: unset; // prevents beacon from jumping on form save via .o-form-saving
```

Comment:
Can you check if this looks good in IE. 
https://caniuse.com/?search=unset

Reply:
I checked and it has no effect, meaning this does not work for ie, I mentioned this in my PR description.